tests/test_core.py

```diff
@@ -0,0 +1,39 @@
+from fastai.core import partition
+import pytest
```

Comment:
I've put this import as the first line by convention, but I'm not gonna insist on that vs just mention it :)

Reply:
no worries i don't mind fixing that